client/src/components/Menu/Menu.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Menu from './Menu';
+
+function handleClick(e) {
+    e.preventDefault();
+    alert("Open me baby!")
+}
+
+storiesOf('Menu', module)
+    .add('Menu', () => <Menu label="Hamburger" type="hammy" onClick={handleClick}>
+        <div className="burgers"></div>
+        <div className="burgers"></div>
+        <div className="burgers"></div>
+    </Menu>)
```

Comment:
Can you move the bars to be part of the component, rather than being children here? Child props are really helpful when we want our component to be able to render JSX from the component's users. Here, we want to be able to define a hamburger and have the rest be taken care of by the component.